shell/src/main/java/org/apache/accumulo/shell/commands/ListScansCommand.java

```diff
@@ -42,21 +45,23 @@ public String description() {
   public int execute(final String fullCommand, final CommandLine cl, final Shell shellState)
       throws Exception {
 
-    List<String> tservers;
-
     final InstanceOperations instanceOps = shellState.getAccumuloClient().instanceOperations();
-
     final boolean paginate = !cl.hasOption(disablePaginationOpt.getOpt());
+    final Set<ServerId> servers = new HashSet<>();
 
     if (cl.hasOption(tserverOption.getOpt())) {
-      tservers = new ArrayList<>();
-      tservers.add(cl.getOptionValue(tserverOption.getOpt()));
+      String serverAddress = cl.getOptionValue(tserverOption.getOpt());
```

Comment:
It seems like this change could be made in 2.1 to improve this command.

Reply:
The 2.1 ListScansCommand has the ability for the user to specify a specific server.